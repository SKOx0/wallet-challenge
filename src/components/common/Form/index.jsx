import styled from 'react-emotion';

export default styled('form')`
  height: ${(props) => props.height};
  width: ${(props) => props.width};
  background-color: ${(props) => props.bgColor};;
`;
