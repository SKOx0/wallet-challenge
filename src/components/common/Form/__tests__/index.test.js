import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Form from '../index';

describe('<Form />', () => {
  it('deve ser igual ao snapshot, com altura, largura, bg e espaÃ§amentos', () => {
    const renderedComponent = shallow(<Form height="500px" width="100px#" bgColor="#ffffff" paddding="20px" />);

    expect(toJson(renderedComponent)).toMatchSnapshot();
  });
});
