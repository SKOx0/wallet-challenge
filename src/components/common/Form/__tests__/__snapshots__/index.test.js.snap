// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Form /> deve ser igual ao snapshot, com altura, largura, bg e espa√ßamentos 1`] = `
.emotion-0 {
  height: 500px;
  width: 100px#;
  background-color: #ffffff;
}

<form
  className="emotion-0"
  height="500px"
  width="100px#"
/>
`;
